src/common/regions.test.ts

```diff
@@ -11,15 +11,69 @@ describe('regions', () => {
     expect(counties).toHaveLength(3224);
   });
 
-  test('findByFipsCode returns a state for a state FIPS', () => {
-    const dc = regions.findByFipsCode('11');
-    expect(dc).toBeTruthy();
-    expect(dc?.regionType).toBe(RegionType.STATE);
+  describe('findByFipsCode', () => {
+    test('returns a state when passing a valid state FIPS', () => {
+      const dc = regions.findByFipsCode('11');
+      expect(dc).toBeTruthy();
+      expect(dc?.regionType).toBe(RegionType.STATE);
+    });
+
+    test('returns a county when passing a valid county FIPS', () => {
+      const county = regions.findByFipsCode('53033');
+      expect(county).toBeTruthy();
```

Comment:
nit: I think `.not.toBe(null)` would better capture the intention of these tests and better complement the tests that *are* expecting null and therefore use `.toBe(null)`.  Feel free to disagree and punt though.